tools/packaging/rpm/istio/istio.spec

```diff
@@ -37,9 +37,7 @@ Source0:        istio.tar.gz
 #Source1:        istiorc
 #Source2:        buildinfo
 Source3:        istio-start.sh
-Source4:        istio-node-agent-start.sh
 Source5:        istio.service
-Source6:        istio-auth-node-agent.service
 
 # e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
```

Comment:
I know you didn't write this original packaging spec.

I answered some questions on stack overflow about building RPMS on RHEL 7.3 on PPC.

Does this build and work on ppc64 architectures?

See stack overflow question and answer here:

https://stackoverflow.com/questions/60394286/gcc-woverloaded-virtual-error-on-powerpc-ppc64le/60443777#60443777

and here:
https://stackoverflow.com/questions/60392262/bazel-tests-fail-on-powerpc-ppc64le-architecture/60695092#60695092